crates/gourgeist/src/bare.rs

```diff
@@ -58,7 +61,11 @@ pub struct VenvPaths {
 }
 
 /// Write all the files that belong to a venv without any packages installed.
-pub fn create_bare_venv(location: &Utf8Path, interpreter: &Interpreter) -> io::Result<VenvPaths> {
+pub fn create_bare_venv(
+    location: &Utf8Path,
+    interpreter: &Interpreter,
+    prompt: Option<&str>,
```

Comment:
Nit: It could make sense to define a custom `Prompt` enum instead of using `Option` that is explicit about the `.` meaning:

```rust
enum Prompt {
	// Use the name of the directory containing the `.venv` as prompt
	DirectoryName,

	// Use this fixed name
	Static(Strring),

	// Don't use a prompt
	None
}